### ISSUE TITLE
ENH: Gallery

### ISSUE THREAD

**Issue by jonas-eschle** [scikit-hep/mplhep#405]
I think it would be useful to have a gallery of different plot styles for comparison and for users to choose from.

Should we add a "Gallery" in the docs? And then in the doc making create the plots?

**Comment by andrzejnovak** [comment]
Indeed, I think this would be helpful, some examples are in the linked pyhep/agc demos, but it's not easily visible. I didn't have time yet to update to the newer style docs that eg hist/awkward have, but that would probably  be a good time to do it.

**Comment by jonas-eschle** [comment]
Ah, you mean to use the pydata_sphinx_theme? Yeah, that could be a good occasion, but that can be a smooth transition.
What do you think, adding another top level menu entry named "Gallery"? Or anything else?

I can get the pydata theme up meanwhile
